migrations/20170310165323-add-notes-to-donations-migration.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+module.exports = {
+  up: function (queryInterface, Sequelize) {
+
+    return queryInterface.addColumn('Donations', 'notes', {
```

Comment:
what's the rationale for notes in the context of a donation?
we used to use description.
we use notes as private instructions to the host to reimburse an expense